source/_static/code/developer/plugins/component-content-type/craftercms-plugin.yaml

```diff
@@ -0,0 +1,43 @@
+# This file describes a plugin for use in Crafter CMS
+
+# The version of the format for this file
+descriptorVersion: 2
+
+# Describe the plugin
+plugin:
+  type: site
+  id: org.craftercms.plugin.mycomponent
+  name: My Component Site Plugin Example
+  tags:
+    - test
+  version:
+    major: 4
+    minor: 0
+    patch: 0
+  description: My simple component content type site plugin
+  website:
+    name: Component Content Type Site Plugin Example
+    url: https://craftersoftware.com
+  media:
+    screenshots:
+      - title: Crafter CMS
+        description: Crafter CMS Example Component Plugin
+        url: "https://raw.githubusercontent.com/craftercms/site-plugin-example/master/.crafter/logo.svg"
```

Comment:
This will change to `.../master/static-assets/.crafter/screenshots/default.png`

Reply:
We should also document that this particular path is the default path for Crafter CMS to look for a default representative image of a plugin or site (see the new Create Site dialog with the thumbnails).